crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -834,6 +863,9 @@ impl SpecializedMeshPipeline for MeshPipeline {
             shader_defs.push("LOAD_PREPASS_NORMALS".into());
         }
 
+        #[cfg(all(feature = "webgl", target_arch = "wasm32"))]
+        shader_defs.push("WEBGL".into());
```

Comment:
Did we not already have this as part of the preset shaderdefs passed to all shaders? Also, should be WEBGL2, that's what it's called for bevy's users.